src/vmm/src/logger/metrics.rs

```diff
@@ -78,6 +78,26 @@ use crate::devices::virtio::vhost_user_metrics;
 use crate::devices::virtio::virtio_block::metrics as block_metrics;
 use crate::devices::virtio::vsock::metrics as vsock_metrics;
 
+#[macro_export]
+/// records aggregate (min/max/sum) for the given $metrics
```

Comment:
sample_count/number of kvm exits for IO and MMIO vm exits is covered by:
```
"exit_io_in",
"exit_io_out",
"exit_mmio_read",
"exit_mmio_write",
```
Count of other vm exits for events like shutdown/hlt/errors are covered by existing "failures" metric.
The only vm exit for which sample_count is not covered is system event reset/shutdown but that should be fine since they are not failures and the vm is terminated anyways.

Reply:
Thanks for the explanation... could we document that in the code, or should that info be in metrics.md?